gensim/models/word2vec.py

```diff
@@ -71,6 +71,7 @@
 import logging
 import sys
 import os
+import warnings
```

Comment:
Rather than using the `warnings` module, these advisories should work fine as more simple `logger.warn()` messages.